test/getMessageTopic.test.ts

```diff
@@ -0,0 +1,63 @@
+import '../index';
+
+describe('get message topic', function () {
+  it('match identical destination and topic', function () {
+    const destinationName = 'dbupdate/_monitor/_area/123/status';
+    const callbackDict = {
+      'dbupdate/_monitor/_area/123/status': () => {},
+      'placeholder/+': () => {},
+      'dbupdate/placeholder/_monitor/two': () => {},
+    };
+
+    const topic = ClearBlade.getMessageTopic(destinationName, callbackDict);
+
+    expect(topic).toEqual('dbupdate/_monitor/_area/123/status');
```

Comment:
No need to change anything but I wonder if we would ever want to match multiple topics. In that case, getMessageTopic would return an array of matches and then we would iterate through and call each callback